bundles/AdminBundle/DependencyInjection/Configuration.php

```diff
@@ -54,6 +54,15 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->end()
                 ->end()
             ->end()
+            ->arrayNode('admin_csp_header')
+                ->addDefaultsIfNotSet()
+                ->children()
+                    ->booleanNode('enabled')
```

Comment:
We could use `->canBeEnabled()` here instead.

Reply:
done